cyences_app_for_splunk/default/macros.conf

```diff
@@ -1040,6 +1045,11 @@ iseval = 0
 definition = search *
 iseval = 0
 
+[cs_network_indexes]
```

Comment:
Let's use this macro everywhere we are using network related stuff.

Reply:
For network, we are using "Network Traffic" datamodel and index used from CIM app `cim_Network_Traffic_indexes`

Can we use it directly?

Reply:
Yes, but we are not using keywork **cim** anymore right??

Reply:
yes but the network traffic model we are using from CIM app so all alerts internally using the `cim_Network_Traffic_indexes` macro.

Reply:
ohh okay, that macro is not present in Cyences then? If not present then it is not recommend to change from our app